run_update.py

```diff
@@ -1197,6 +1204,27 @@ def update_attendance(session, organization_name, attendance_dict):
     return existing_attendance
 
 
+def get_logo(org_info):
+    ''' get an organization's logo, looking first at Github (project lists url) '''
```

Comment:
It's first checking for an explicit `logo_url` right?

Reply:
Good point.